src/odemis/gui/doc/sparc2_streakcam.html

```diff
@@ -1,7 +1,7 @@
       <font size="2">
         <b><font size="2.5">Streak camera hardware alignment</font></b>
           <ol>
-            <li>At all time limit the pixel intensity to max. 1000 cts.</li>
+            <li>Limit the pixel intensity to max. 1000 cts/100msec!!!</li>
```

Comment:
Triple "!" is not needed. In general I tend to think that there is very little reason to put any exclamation marks in a documentation ;-) Also the standard way to write "milliseconds" is "ms".

Reply:
only one line - so many comments :D